src/PrestaShopBundle/Resources/views/Admin/Common/Grid/Columns/Content/highlighted.html.twig

```diff
@@ -0,0 +1,34 @@
+{#**
+ * 2007-2019 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+
+<div class="text-right">
+  {% if record[column.options.is_highlighted_field]%}
+    <span class="badge badge-success rounded">
```

Comment:
So we only allow `badge-success` to be used ? What about we define at least some const like SUCCESS_HIGHLIGHT, WARNING_HIGHLIGHT, DANGER_HIGHLIGHT and map the css class to be used on this ?
Would'nt it make it more reusable ?
```
{% if record[column.options.highlight_level] == 'success' %}
    <span class="badge badge-success rounded">
{% if record[column.options.highlight_level] == 'warning' %}
    <span class="badge badge-warning rounded">
{% else %}
```

Reply:
See https://github.com/PrestaShop/PrestaShop/pull/14098/commits/29d865ad82d792a3e752d5143b38c55b55448174